modules/local/seqtk_size.nf

```diff
@@ -17,7 +17,7 @@ process SEQTK_SIZE{
     def prefix = task.ext.prefix ?: "${meta.id}"
     output = "${meta.id}_basecounts.txt"
     """
-    seqtk size ${reads.join(" ")} > ${output}
+    zcat ${reads.join(" ")} | seqtk size - > ${output}
```

Comment:
Fix to support paired-end data since `seqtk size` only accepts a single fastq file as input.

Reply:
Thank you so much for identifying this, it could have become a much larger future issue if you had not caught it!